docs/changelog-nl.rst

```diff
@@ -9,6 +9,211 @@ Changelog (NL)
     found :ref:`here <changelog>`.
 
+3.2.0 "Nimma" (2025-07-10)
+==========================
+
+Open Formulieren 3.2.0 is een feature release.
+
+.. epigraph::
+   "Nimma" is een informele, liefhebbende bijnaam voor een van de oudste
+   steden in Nederland: Nijmegen. De bijnaam wordt vaak gebruikt door de lokale
+   bevolking en geeft een gevoel weer van trots, solidariteit en persoonlijke identiteit.
+   Natuurlijk zijn we ook trots dat Nijmegen bijdraagt aan Open Forms.
+
+Deze release bevat wijzigingen van de alpha releases en oplossingen tot aan de
+stable versie. Lees de release notes aandachtig voor het upgraden naar versie 3.2.0
+en volg de instructies zoals hieronder beschreven:
+
+Upgrade procedure
+-----------------
+
+.. warning::
+
+   The Camunda registratie backend zal verwijderd worden in Open Formulieren 4.0. Er is geen vervanging
+   gepland - neem contact op als je afhankelijk van deze plugin bent.
+
+.. warning::
+
+   De manier waarop data wordt gegenereerd via de generieke JSON registratie plugin is
+   aangepast. Indien er conflicten optreden tussen statische, component, en user-gedefinieerde
+   variabelen worden de statische variabelen gebruikt. Voorheen werden in dit soort gevallen
+   de component en user-gedefinieerde gebruikt. De validatie zorgt ervoor dat het gebruik van
+   al gedefinieerde variable, via de statische variabelen, niet mogelijk is. Dit geldt echter
+   niet voor oudere formulieren of nieuw toegevoegde statische variabelen.
+
+Belangrijkste verbeteringen
+---------------------------
+
+**🔊 Verbeterde logging**
+
+    De logging is verbeterd voor betere integratie met observability tooling zoals Grafana.
+
+**🛂 Authenticatie d.m.v Yivi en eIDAS**
+
+    Ondersteuning is toegevoegd voor `Yivi <https://yivi.app/>`_ en `eIDAS <https://en.wikipedia.org/wiki/EIDAS>`_
+    authenticatie d.m.v OpenID Connect Protocol. Eind gebruikers kunnen nu kiezen welke informatie
+    zij willen delen met Open Formulieren door de ondersteuning voor Yivi authenticatie.
+
+    Door de ondersteuning van eIDAS kunnen europese burgers zonder DigiD (en/of BSN)
+    toegang krijgen tot formulieren die deze manier van authenticatie vereisen.
+
+**👫 Partners component met prefill**
+
+    Het partners component is toegevoegd om informatie zoals initialen, achternaam en
+    geboortedatum van een partner te tonen of aan te leveren.
+
+    Dit component kan vooringevuld worden door het gebruik van de nieuwe familielid prefill plugin.
+    De familielid prefill plugin kan informatie opvragen vanuit "Haal Centraal BRP personen bevragen"
+    (version 2) of "StUF-BG" (version 3.1).
+
+**📝 JSON schema generatie**
+
+    Het genereren van een JSON schema van een formulier is sinds deze release mogelijk.
+    Het beschrijft de data van een ingediend formulier van alle gebruiker-gedefinieerde
+    en component variabelen en kan gegenereerd worden voor de Generieke JSON en Objecten API
```

Comment:
some more recurring terminilogy-issues here, probably best to find-and replace a bit!

Reply:
See a482eeec62dd7b598234b7227cc41f90d46e275e